app/api/leaderboard/functions.ts

```diff
@@ -113,3 +114,42 @@ export default async function fetchGitHubReleases(
     )
     .slice(0, sliceLimit);
 }
+
+interface RepositoriesResponse {
+  organization: {
+    repositories: {
+      nodes: {
+        name: string;
+      }[];
+    };
+  };
+}
+
+export async function fetchAllReposName() {
+  const result: RepositoriesResponse = await octokit.graphql.paginate(
+    `
+      query paginate($cursor: String, $organization: String!) {
+        organization(login: $organization) {
+          repositories(first: 10, after: $cursor, orderBy: { field: STARGAZERS, direction: DESC }) {
+            nodes {
+              name
+            }
+            pageInfo {
+              hasNextPage
+              endCursor
+            }
+          }
+        }
+      }
+      `,
+    {
+      organization: "coronasafe",
```

Comment:
Lets not hard code the org. name since this would be deployed by other orgs. too.

Reply:
Ohh, sorry i missed that